src/components/Header/WalletDisplay.tsx

```diff
@@ -24,35 +24,40 @@ type TProps = {
 
 export const Network: React.FC<TProps> = (props) => {
   const { network } = props;
+
+  let networkTitle = "";
+
+  switch (network) {
+    case ENetwork.RINKEBY:
+      networkTitle = "Rinkeby Testnet";
+      break;
+    case ENetwork.POLYGON:
+      networkTitle = "Polygon";
+      break;
+    case ENetwork.MUMBAI:
+      networkTitle = "Polygon Testnet";
+      break;
+    case ENetwork.UNSUPPORTED:
+      networkTitle = "Unsupported Chain";
+      break;
+    default:
+      networkTitle = "Invalid network state!"; // Not sure if this case will ever be reached, but I 'm preserving it
```

Comment:
Yea exactly I was looking at this the other day.

My thinking is that with a switch on an enum the default case/condition will only be met if you've not handled one of the enum's possible values so maybe the only thing to do is throw an error.